frontend/server/src/DAO/Schools.php

```diff
@@ -145,7 +145,9 @@ public static function getMonthlySolvedProblemsCount(
                     AND sub.time < su.time
             )
         GROUP BY
-            YEAR(su.time), MONTH(su.time);';
+            YEAR(su.time), MONTH(su.time)
+        ORDER BY
+            YEAR(su.time) ASC, MONTH(su.time) ASC;';
```

Comment:
nit: puedes ponerlos uno por línea?

```suggestion
            YEAR(su.time),
            MONTH(su.time)
	ORDER BY
            YEAR(su.time) ASC,
            MONTH(su.time) ASC;';
```